.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
```

Comment:
#### `HINT`

:bulb: To avoid making multiple commits while fixing linter errors, try this. :point_down: 

>  :boom: 
> - run [stylelint](https://stylelint.io/) in your local env:
>     - run `npm install stylelint stylelint-config-recommended --save-dev`  (not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
>     - copy [stylelint.config.js](./stylelint.config.js) to the root directory of your project
>     - run `npx stylelint .`
>     - fix linter errors